src/System.Globalization.Extensions/src/System.Globalization.Extensions.csproj

```diff
@@ -13,9 +13,11 @@
     <AssemblyVersion>4.0.1.0</AssemblyVersion>
     <ProjectGuid>{2B96AA10-84C0-4927-8611-8D2474B990E8}</ProjectGuid>
     <PackageTargetFramework Condition="'$(PackageTargetFramework)'==''">dotnet5.4</PackageTargetFramework>
-    <IsPartialFacadeAssembly Condition="'$(TargetGroup)'=='net46'">true</IsPartialFacadeAssembly>
-    <UsePackageTargetRuntimeDefaults>true</UsePackageTargetRuntimeDefaults>
     <NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETPlatform,Version=v5.4</NuGetTargetMoniker>
+    <IsPartialFacadeAssembly Condition="'$(TargetGroup)'=='net46'">true</IsPartialFacadeAssembly>
+    <PackageTargetRuntime Condition="'$(TargetsWindows)' == 'true'">win7</PackageTargetRuntime> 
+    <PackageTargetRuntime Condition="'$(TargetsUnix)' == 'true'">unix</PackageTargetRuntime> 
+    <PackageTargetRuntime Condition="'$(TargetGroup)'=='net46'" />
```

Comment:
Instead can we constrain the above to '$(TargetGroup)' == ''.  We tend to follow the pattern where we make these conditions mutually exclusive rather than sequential.